scripts/list-repos-matching-extensions.js

```diff
@@ -0,0 +1,46 @@
+const { searchByCriteria } = require("../util/github");
+
+module.exports = {
+    description: "list language usage within an organization",
+    options: {
+        organization: {
+            alias: "o",
+            demandOption: true,
+            describe: "the GitHub organization to search for language usage",
+            type: "string"
+        },
+        extensions: {
+            alias: "e",
+            demandOption: true,
+            describe: "a comma seperated list of extensions being searched for",
+            type: "string"
```

Comment:
we should make this an `array` instead of a comma separated string.  example: https://github.com/liatrio/github-scripts/blob/main/scripts/enable-secret-scanning.js#L19

Reply:
this also means that users will have to use this flag multiple times in order to specify different extensions